src/app/Shared/Model/Isolate.model.ts

```diff
@@ -1,3 +1,175 @@
+            ery: {
+                value: string;
+                active: boolean;
+            };
+            etp: {
+                value: string;
+                active: boolean;
+            };
+            fep: {
+                value: string;
+                active: boolean;
+            };
+            fop: {
+                value: string;
+                active: boolean;
+            };
+            fox: {
+                value: string;
+                active: boolean;
+            };
+            fus: {
+                value: string;
+                active: boolean;
+            };
+            f_c: {
+                value: string;
+                active: boolean;
+            };
+            gen: {
+                value: string;
+                active: boolean;
+            };
+            imi: {
+                value: string;
+                active: boolean;
+            };
+            kan: {
+                value: string;
+                active: boolean;
+            };
+            lzd: {
+                value: string;
+                active: boolean;
+            };
+            mero: {
+                value: string;
+                active: boolean;
+            };
+            mup: {
+                value: string;
+                active: boolean;
+            };
+            nal: {
+                value: string;
+                active: boolean;
+            };
+            pen: {
+                value: string;
+                active: boolean;
+            };
+            rif: {
+                value: string;
+                active: boolean;
+            };
+            smx: {
+                value: string;
+                active: boolean;
+            };
+            str: {
+                value: string;
+                active: boolean;
+            };
+            syn: {
+                value: string;
+                active: boolean;
+            };
+            taz: {
+                value: string;
+                active: boolean;
+            };
+            tec: {
+                value: string;
+                active: boolean;
+            };
+            tet: {
+                value: string;
+                active: boolean;
+            };
+            tgc: {
+                value: string;
+                active: boolean;
+            };
+            tia: {
+                value: string;
+                active: boolean;
+            };
+            tmp: {
+                value: string;
+                active: boolean;
+            };
+            trm: {
+                value: string;
+                active: boolean;
+            };
+            t_c: {
+                value: string;
+                active: boolean;
+            };
+            van: {
+                value: string;
+                active: boolean;
+            };
+        };
+    }[];
+}
 
-export type DBentry = Record<DBkey, string>;
+
+export type DBentry = IsolateApiInterface["isolates"];
```

Comment:
Reject: your business logic model should not depend on the api model,  i think your actual app does not interpret all the stuff in the api model? So keep an independent model for your app logic and map the api model to that model (only the data that is necessary and to a format that fits your components inputs)

Reply:
Reject2: your dbentry is no entry anymore, it is a collection of the dbentries, so you should rename it

Reply:
done. 
- Only use desired values from isolate-API